src/formElementPlugins/password/src/ApplicationView.tsx

```diff
@@ -45,7 +52,9 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
       setPassword(e.target.value)
       const responses = { thisResponse: password || '' }
       const customValidation = await validate(validationInternal, validationMessageInternal, {
-        objects: { responses },
+        objects: { responses, currentUser, applicationData },
+        APIfetch: fetch,
```

Comment:
Does this need also binding? Please check with Andrei

Reply:
Don't think so, it's exactly the same evaluator parameters used throughout, and there's no problem with the API calls from the front-end.

Reply:
I think the only time we want to bind is when we are accessing fetch through an object.
i.e.
```JS
const obj = { fetch }
obj.fetch(...) // should be a problem without bind
const { fetch: fetchMe } = object
fetchMe(...) // should work
```